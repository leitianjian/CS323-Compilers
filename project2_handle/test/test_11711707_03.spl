int compare(int x, int y)
{
    if (x > y) {
        return 1;
    } else if (y > x) {
        return -1;
    } else {
        return 0;
    }
}

float compare(int a, int b)
{
    if (b > a) {
        return 1;
    }

    if (b == a) {
        return 0.0;
    }

    return 'a';
}