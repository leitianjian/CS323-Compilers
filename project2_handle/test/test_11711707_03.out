Error type 4 at Line 12: compare function is redefined
Error type 8 at Line 15: function return type is not matched
Error type 8 at Line 22: function return type is not matched
